packages/dds/tree/src/test/feature-libraries/editable-tree/editableTree.spec.ts

```diff
@@ -300,7 +310,16 @@ describe("editable-tree", () => {
             },
         ]);
         const context = getEditableTreeContext(forest);
-        assert.equal((context.unwrappedRoot as EditableTree)["child" as FieldKey], 1);
+        assert(isUnwrappedNode(context.unwrappedRoot));
+        assert.equal(context.unwrappedRoot["child" as FieldKey], 1);
```

Comment:
use `brand`: its a little more type safe:
```suggestion
        assert.equal(context.unwrappedRoot[brand("child")], 1);
```

Same below.

Reply:
I tried, but somehow, ts does not see an index type of EditableTree.